qutebrowser/config/configfiles.py

```diff
@@ -93,9 +94,26 @@ def init_save_manager(self, save_manager):
         """
         save_manager.add_saveable('yaml-config', self._save)
 
+    def __getitem__(self, name):
+        return self._values[name]
+
+    def __setitem__(self, name, value):
+        if self._values.get(name) != value:
```

Comment:
`self._values.get(name)` defaults to `None`, right? So if we'd set a value to `None` which wasn't set before, this wouldn't detect the change, if I'm not mistaken?

I guess you'd either define a `self._UNDEFINED = object()` or so as sentinel object, or just always set the flag here - I mean, if someone uses `:set` and sets it to the value it was already, re-saving the file should be fine?